gadget/install/install.go

```diff
@@ -53,7 +53,7 @@ func deviceFromRole(lv *gadget.LaidOutVolume, role string) (device string, err e
 
 // Run bootstraps the partitions of a device, by either creating
 // missing ones or recreating installed ones.
-func Run(model gadget.Model, gadgetRoot, device string, options Options, observer gadget.ContentObserver) (*InstalledSystemSideData, error) {
+func Run(model gadget.Model, gadgetRoot, kernelRoot, device string, options Options, observer gadget.ContentObserver) (*InstalledSystemSideData, error) {
```

Comment:
I think there may be too many parameters here, can gadgetRoot and kernelRoot and things go into the Options (or maybe rename Options to something like Params, etc. ) ?

Reply:
6 params may be a bit too many indeed. Extending Options sounds ok to me.

Reply:
I think 6 is borderline, having Run(Options)  would also be a bit weird to me

Reply:
I added a TODO for myself to look at this for a followup